build.gradle

```diff
@@ -1,42 +1,27 @@
 buildscript {
-    ext.kotlinVersion = "1.6.21"
     ext.compileVersion = 33
-    ext.buildToolsVersion = "33.0.2"
     ext.minVersion = 14
-    ext.billingVersion = "5.1.0"
-    ext.lifecycleVersion = "2.5.0"
-    ext.testLibrariesVersion = "1.4.0"
-    ext.testJUnitVersion  = "1.1.3"
-    ext.robolectricVersion = "4.10.3"
-    ext.mockkVersion = "1.12.8"
-    ext.assertJVersion = "3.22.0"
-    ext.annotationVersion = "1.3.0"
-    ext.androidxCoreVersion = "1.8.0"
-    ext.mockwebserverVersion = "4.2.0"
-    ext.tinkVersion = "1.8.0"
-    ext.detektVersion = "1.23.0"
-    ext.coroutinesVersion = "1.6.4"
-
-    repositories {
-        mavenCentral()
-        google()
-    }
-    dependencies {
-        classpath "com.vanniktech:gradle-maven-publish-plugin:0.22.0"
-        classpath 'com.android.tools.build:gradle:8.0.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.8.10"
-        classpath "org.jetbrains.kotlinx:kover-gradle-plugin:0.7.0"
-    }
 }
 
 plugins {
-    id "io.gitlab.arturbosch.detekt" version "$detektVersion"
-    id "com.savvasdalkitsis.module-dependency-graph" version "0.9"
+    alias libs.plugins.mavenPublish apply false
+    alias libs.plugins.android.application apply false
+    alias libs.plugins.android.library apply false
+    alias libs.plugins.kotlin.android apply false
+    alias libs.plugins.dokka
+    alias libs.plugins.kover apply false
+    alias libs.plugins.detekt
+    alias libs.plugins.dependencyGraph
+    //    TODO: is this needed?
+    alias libs.plugins.androidx.navigation.safeargs apply false
+    //    TODO: is this needed?
+    alias libs.plugins.kotlin.kapt apply false
+    //    TODO: is this needed?
```

Comment:
Hmm are these `TODO`'s  meant to be removed?

Reply:
Looks like we only use parcelization in the `public` and `feature/amazon` modules, so I believe it should only be needed in those...